metaswitch/common/mib.py

```diff
@@ -40,10 +40,13 @@ def get_all_stats(self, columns):
         """
         # Add INDEX to the list of fields to retrieve and parse - we
         # need this in order to check whether a node is a statistic or not.
-        if "INDEX" not in columns:
-            columns.append("INDEX")
+        # To do this we use a copy of columns to keep ownership of columns in
+        # the calling script.

```

Comment:
This comment makes assumptions about how this code is called - that's bad. It's better to explain why this code needs to own the list of columns, i.e. because it needs to edit it.